SPECS/kernel/config

```diff
@@ -12,7 +12,6 @@ CONFIG_LD_IS_BFD=y
 CONFIG_LD_VERSION=23700
 CONFIG_LLD_VERSION=0
 CONFIG_CC_CAN_LINK=y
-CONFIG_CC_CAN_LINK_STATIC=y
```

Comment:
Modifying the kernel config will take more changes than that, I'm afraid. For example, take a look at #3826.

I'm not sure, if we want to bundle them up into this PR. We usually try to keep kernel changes in a separate PR.

Reply:
This would need to be a separate change.  Why is this needed?

Reply:
I had a chat with @bossmc on this. The glibc change alters this `CONFIG_CC_CAN_LINK_STATIC` Kconfig, which is dynamically detected during kernel build. The config is NOT meant to be user-settable. The only spots this config is used is when `CONFIG_BPFILTER_UMH` or `CONFIG_BPF_PRELOAD_UMD` is enabled, both of which are not active/enabled in our kernels. Therefore there is no resulting change in the underlying kernel builds.

So IMO we can consider this change as a no-patch situation and forego the dash-roll